tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/metrics/AllMetricsBean.java

```diff
@@ -38,9 +39,13 @@
     @Bulkhead(3)
     @Timeout(value = 1000, unit = ChronoUnit.MILLIS)
     @CircuitBreaker(failureRatio = 1.0, requestVolumeThreshold = 20)
+    @Fallback(fallbackMethod = "doFallback")
```

Comment:
The TCK should also cover the case when a fallback handler is used: `@Fallback(MyFallbackHandler.class)`

Reply:
Done. I've added as a second test in `FallbackMetricTest`.